collectors/wallet_collector.go

```diff
@@ -169,4 +183,30 @@ func (u *WalletCollector) Collect(ch chan<- prometheus.Metric) {
 		u.unconfirmedBalanceDesc, prometheus.GaugeValue,
 		float64(walletBal.Unconfirmed),
 	)
+
+	accounts, err := u.lnd.WalletKit.ListAccounts(context.Background(), "", 0)
```

Comment:
nit: wrap at 80 chars

Reply:
isn't it already below 80? (vim shows :78)

Reply:
mine shows 83. Is your tab size set to 8 spaces?